### DISCUSSION TITLE
external service

### DISCUSSION THREAD

**Discussion by Ayaz-Tanzeem** [istio/istio#45430] [General]
I am trying to configure istio to connect to a service running outside my k8s mesh. I configured a service entry and a virtual service for the same. But I am unable to connect to it

**Comment by sovitacharya** [comment]
I have figured out to do the same for endpoints  with IP , but when I need to proxy my request to an https endpoint using istio gateway , I am unable to process the same.

**Reply by sovitacharya** [reply]
@howardjohn 
Can you help us with this?

**Reply by Ayaz-Tanzeem** [reply]
were you able to connect to a service running outside k8s mesh via istio?

**Reply by sovitacharya** [reply]
Yes @Ayaz-Tanzeem  i was able to do so , but when I am trying to do the same for an https endpoint I am not able to do so .
